html-css/week2/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Good you made a PR, it's easier to check. And great work with the website, it's fully responsive 👍